### ISSUE TITLE
Would you release scripts for ILSVRC2016?

### ISSUE THREAD

**Issue by wk910930** [weiliu89/caffe#303]
Dear Wei,

I am training SSD300x300 on ILSVRC2016 and would like to repeat your result (mAP=43.4) on paper. However, I found training loss is very unstable (e.g., I can easily meet NaN loss even with base_lr=0.0005). The script I am using is modified from ssd_pascal.py. Apparently, there are much details I am missing.

Would you release scripts for ILSVRC2016?

Thanks!

**Comment by weiliu89** [comment]
I will provide a pretrained model with python script later.
